IFC4x3/Properties/f/FaultDisplacement_1SqeJ7wlr3KhX7hE0uIsj3/Documentation.md

```diff
@@ -0,0 +1 @@
+Contains a text summarising the displacement across the ShearDisplacementStructure.
```

Comment:
```suggestion
Contains a text summarising the displacement across the ShearDisplacementStructure.

> NOTE: Definition according to OGC Geoscience Markup Language 4.1 (GeoSciML) (8.9.8.8 displacement)```

Reply:
Editing proposal "Summary describing the displacement across ..."

@larswik can you amend your proposal?